dist/neo.blockchain.db.js

```diff
@@ -95,50 +96,52 @@ module.exports = function (network) {
      * of balance discretization.
      * @returns Promise.<Array> An array containing the balances of an address.
      */
-    this.getBalance = ({ address, assets = node.assets, blockAge = 1 }) =>
-    new Promise((resolve, reject) => {
-      module.addresses.findOne({'address': address})
-        .exec((err, res) => {
-          if (err) reject(err)
-
-          // If the address is not found in the database, its new...So add it and retry.
-          if (!res) {
-            module.addresses({'address': address, 'type': 'c', 'assets': []})
-              .save((res) => {
-                node.getBalance(address)
-                .then((res) => {
-                  resolve(res)
+    this.getBalance = ({ address, assets = node.assets, blockAge = 1 }) => {
+      return new Promise((resolve, reject) => {
+        module.addresses.findOne({ address })
+          .exec((err, res) => {
+            if (err) reject(err)
+
+            // If the address is not found in the database, its new... So add it and retry.
+            if (!res) {
+              module.addresses({ address, type: 'c', assets: [] })
+                .save((res) => {
+                  node.getBalance(address)
+                    .then((res) => {
+                      resolve(res)
+                    })
                 })
-              })
-          } else {
-            // Sort the assets into 'current' and 'needs update'
-            var parts = _.partition(res.assets, (asset) =>
-              (node.index - asset.index) >= blockAge
-            )
-            // If there is an asset list discripancy, scan for missing assets to update.
-            // This mechanic is used to automatically add new asset support as
-            // an asset it appears in a transaction.
-            if (res.assets.length !== node.assets.length) {
-              var included = _.map(res.assets, 'asset')
-              node.assets.forEach((asset) => {
-                if (included.indexOf(asset.asset) === -1) {
-                  parts[0].push({
-                    'asset': asset.asset
-                  })
-                }
-              })
-            }
+            } else {
+              // Sort the assets into 'current' and 'needs update'
+              let parts = _.partition(res.assets, (asset) =>
+                (node.index - asset.index) >= blockAge
+              )
+              /**
+               * If there is an asset list discripancy, scan for missing assets to update.
+               * This mechanic is used to automatically add new asset support as
+               * an asset it appears in a transaction.
+               */
+              if (res.assets.length !== node.assets.length) {
+                let included = _.map(res.assets, 'asset')
+                node.assets.forEach((asset) => {
```

Comment:
If the only property we use from `asset` is `.asset` then maybe we should destructure the the first argument.

```JavaScript
node.assets.forEach(({ asset }) => {
  if (!included.includes(asset)) {
    parts[0].push({ asset })
  }
})
```